benchmark/src/test/scala/code/chymyst/benchmark/MergesortSpec.scala

```diff
@@ -46,54 +47,54 @@ class MergesortSpec extends FlatSpec with Matchers {
     }
     mergeRec(0,0,0)
     if (wantToLog) println(s"${System.currentTimeMillis} finished merging #$id")
-    result.toArray
+    result.toIndexedSeq
   }
 
-  def performMergeSort[T : Ordering : ClassTag](array: Array[T], threads: Int = 8): Array[T] = {
+  def performMergeSort[T : Ordering](array: Coll[T], threads: Int = 8): Coll[T] = {
```

Comment:
I just wanted to make it easier to change the collection type everywhere, in case it's necessary. That's the only reason for `Coll` as a separate type.